app/services/workflow_errors_reporter.rb

```diff
@@ -0,0 +1,31 @@
+require 'faraday'
+# This class will send a put ReST call to the preservationWF via faraday. This also parses
+# out a shorter error message, that only includes the moab validation error.
+class WorkflowErrorsReporter
+
+  def self.update_workflow(druid, error_message)
+    error_message.select { |error| request(druid, 'moab-valid', moab_error(error)) if error.key?(PreservedObjectHandlerResults::INVALID_MOAB) }
+  end
+
+  private_class_method def self.conn
+    if Settings.workflow_services.url.present?
+      @connection ||= Faraday.new(url: Settings.workflow_services.url)
+      Rails.logger.log(warn, 'no workflow hookup - assume you are in test or dev environment')
```

Comment:
I think we want the logic to say:

```ruby
private_class_method def self.conn
  @@connection || = begin
    if Settings.workflow_services.url.present?
       Faraday.new(url: Settings.workflow_services.url)
    else
      'no workflow hookup - assume you are in test or dev environment'
    end
  end
end
```

Would like the above to make the connection object ONE time, as a CLASS constant, and would like to only error ONE time if there is no url.  However, if we set it to nil when there is no url, then it will try each time to make the connection.  And if we don't set it to nil when we have none, then we have to check in the `request` method to see if it's a connection or not ...

Note: the logging can be simpler:

```ruby
Rails.logger.warn(msg_string)
```